.circleci/config.yml

```diff
@@ -704,8 +810,21 @@ jobs:
             # Fail this step depending on the success to trigger a rerun of this step together with others in case of a "rerun failed"
             [[ $OUTCOME = "success" ]] && exit 0 || exit 1
 
+
+####################
+## Workflow setup ##
+####################
 workflows:
-  main:
+  acceptance-only-run:
```

Comment:
Awesome! The separation into different workflows which are executed based on a clear condition makes it very transparent what's going on! 🚀